etc/docker/dev/docker-compose-storage-alldb.yml

```diff
@@ -1,129 +1,106 @@
-version: "2"
 services:
+  rucio:
+    image: docker.io/rucio/rucio-dev:py3
+    extra_hosts:
+      - "ruciomy5:127.0.0.1"
+      - "ruciomy8:127.0.0.1"
+      - "oracle:127.0.0.1"
+      - "ruciodb:127.0.0.1"
+      - "graphite:127.0.0.1"
+      - "fts:127.0.0.1"
+      - "ftsdb:127.0.0.1"
+      - "xrd1:127.0.0.1"
+      - "xrd2:127.0.0.1"
+      - "xrd3:127.0.0.1"
+      - "minio:127.0.0.1"
+    volumes:
+      - ../../../tools:/opt/rucio/tools:Z
+      - ../../../bin:/opt/rucio/bin:Z
+      - ../../../lib:/opt/rucio/lib:Z
+    environment:
+      - X509_USER_CERT=/opt/rucio/etc/usercert.pem
+      - X509_USER_KEY=/opt/rucio/etc/userkey.pem
+      - RDBMS=postgres11
   ruciomy5:
-    image: mysql:5
-    hostname: ruciomy5
+    image: docker.io/mysql:5
+    network_mode: "service:rucio"
```

Comment:
A thing to be taken into consideration: because the network binds to the service "rucio", re-creating only this container breaks connectivity between it and other containers. It is necessary to re-create the whole docker-compose environment if the rucio container is restarted.

Reply:
I can avoid this by adding an infra container and attach everything to the infra container. Would that be better?

Reply:
In my local config it works correctly without any network_mode at all. And without any extra_hosts. By default all containers are put into the same network and can access each-other by their name = hostname. Any reason why we want to diverge from the default behavior ?

Reply:
The answer to that is in the PR description

Reply:
ACK. it's thus due to the podman restrictions. I have no preferences at all, I'll have to use my own docker-compose file override anyway because i'll customize it for my personal workflow. Good for me to leave it as-is